pyproject.toml

```
@@ -1,3 +1,49 @@
+[project]
+name = "voicevox_engine"
+version = "0.24.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = "==3.11.9"
+dependencies = [
+  "fastapi-slim>=0.115.5,<0.116.0",
+  "jinja2>=3.1.3,<4.0.0",           # NOTE: required by fastapi.templating.Jinja2Templates (fastapi-slim's unmanaged dependency)
+  "numpy>=2.2.3,<3.0.0",
+  "platformdirs>=4.2.0,<5.0.0",
+  "pydantic>=2.7.3,<3.0.0",
+  "pyopenjtalk",
+  "python-multipart==0.0.20",       # NOTE: required by fastapi.Form (fastapi-slim's unmanaged dependency)
```

uvを実行すると、コメントの空白が自動フォーマットされました。（uv 0.6.14）

```suggestion
  "jinja2>=3.1.3,<4.0.0", # NOTE: required by fastapi.templating.Jinja2Templates (fastapi-slim's unmanaged dependency)
  "numpy>=2.2.3,<3.0.0",
  "platformdirs>=4.2.0,<5.0.0",
  "pydantic>=2.7.3,<3.0.0",
  "pyopenjtalk",
  "python-multipart==0.0.20", # NOTE: required by fastapi.Form (fastapi-slim's unmanaged dependency)
```

---

`uv add` によるパッケージ追加でも同様のコメント自動フォーマットが発生しました。  
そのままだと他の PR にこの変更が紛れ込むので、@takana-v さんの suggested change を取り込むのが良いと考えます。

---

@takana-v さん、 @tarepan さん、レビューありがとうございます。

こちらは私がVSCodeに入れているtomlの拡張機能のオートフォーマットが原因でした。
以下コミットにて修正いたしました。

d60ba60cd87dfd9ef6aa9fb3c5415db4c0d21329